doc/man3/UI_new.pod

```diff
@@ -177,11 +180,15 @@ UI_get_method() returns the UI method associated with a given UI.
 
 UI_set_method() changes the UI method associated with a given UI.
 
-UI_OpenSSL() is the default OpenSSL UI method for prompting
-passphrases on the command line.
+=head1 NOTES
 
-UI_null() is a UI method that does nothing.  Its use is to avoid
-getting internal defaults for passed UI_METHOD pointers.
+The resulting strings that the built in method UI_OpenSSL() generate
+are encoded according to the current locale or (forWindows) code page.
+For applications having different demands, these strings need to be
+converted appropriately by the caller.
+For Windows, it's possible to get an UTF8 encoded string by defining
```

Comment:
I'm not sure if I'd leave out the paragraph about Windows and UTF-8. Or at least try to make the caveats clearer — at first reading it seems to advocate messing with the environment variable at runtime, which has potential issues with threading. Maybe phrase it as *"if the `OPENSSL_WIN32_UTF8` environment variable is set, the built-in method `UI_OpenSSL()` will produce UTF-8 encoded strings"*?